understanding/20/labels-or-instructions.html

```diff
@@ -39,6 +39,9 @@ <h2>Intent of Labels or Instructions</h2>
          The goal is to make certain that enough information is provided for the user to accomplish
          the task without undue confusion or navigation.
       </p>
+
+      <p>Labels of form controls are usually text-based. In some cases, images can serve as descriptive labels without additional text. 
+        In these cases, authors should ensure that the image and its use as a label (in context) are widely understood.</p>
```

Comment:
I feel like this "widely understood" sentence would probably be better in Headings and Labels. Although it is goes beyond a simple discussion on descriptive text, the qualitative concepts of Headings and Labels seems like a better fit than the basic question of 3.3.2 which is: is there a label for the input?

Some task force members ended up having a long post-meeting discussion on the nuances of the telephone example in the original issue. It would be good to try to capture some of those concepts in our guidance, in some way.

Reply:
agree, the qualitative part should be moved to headings and labels (and, at a stretch, then cross-referenced from here)